Tools/autotest/pysim/util.py

```diff
@@ -64,25 +64,45 @@ def deltree(path):
 def relwaf():
     return "./modules/waf/waf-light"
 
-def build_SIL(build_target, j=None, debug=False):
-    '''build desktop SIL'''
-
-    # first configure
-    cmd_configure = [relwaf(), "configure", "--board", 'sitl']
+def waf_configure(board, j=None, debug=False):
+    cmd_configure = [relwaf(), "configure", "--board", board]
     if debug:
         cmd_configure.append('--debug')
     if j is not None:
         cmd_configure.extend(['-j', str(j)])
     run_cmd(cmd_configure, dir=topdir(), checkfail=True)
 
-    # then clean
+def waf_clean():
     run_cmd([relwaf(), "clean"], dir=topdir(), checkfail=True)
 
+def build_SIL(build_target, j=None, debug=False, board='sitl'):
+    '''build desktop SIL'''
+
+    # first configure
+    waf_configure(board, j=j, debug=debug)
+
+    # then clean
+    waf_clean()
+
     # then build
-    cmd_make = [relwaf(), "build", "--target", build_target]
+    cmd_make = [relwaf(), 'build', "--target", build_target]
```

Comment:
No, they are equivalent. I suggest running pep8 and check if it recommends anything